web/docs/operators/azure-log-analytics.md

```diff
@@ -0,0 +1,75 @@
+---
+sidebar_custom_props:
+  operator:
+    source: false
+    sink: true
+---
+
+# azure-log-analytics
+
+Sends events via the [Microsoft Azure Logs Ingestion
+API](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/logs-ingestion-api-overview).
+
+## Synopsis
+
+```
+azure-log-analytics --tenant-id <tenant-id> --client-id <client-id>
+--client-secret <client-secret> --dce <data-collection-endpoint> --dcr
+<data-collection-rule-id> --table <table-name> [--batch-size <batch-size>]
+```
+
+## Description
+
+The `azure-log-analytics` operator makes it possible to upload events to
+[supported
+tables](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/logs-ingestion-api-overview#supported-tables)
+or to [custom tables](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/create-custom-table?tabs=azure-portal-1%2Cazure-portal-2%2Cazure-portal-3#create-a-custom-table)
+in Microsoft Azure.
+
+The operator handles access token retrievals by itself and updates that token
+automatically, if needed.
+
+### `<tenant-id>`
```

Comment:
If you take a look at other docs, we always write the full option here: ` --tenant-id <tenant-id>`.

Reply:
Fixed in https://github.com/tenzir/tenzir/pull/4281/commits/0978fc38aa2668e38bb41980a7ac94c8ba7b491d